libc/src/math/generic/cos.cpp

```diff
@@ -99,7 +99,7 @@ LLVM_LIBC_FUNCTION(double, cos, (double x)) {
   // Fast look up version, but needs 256-entry table.
   // -sin(k * pi/128) = sin((k + 128) * pi/128)
   // cos(k * pi/128) = sin(k * pi/128 + pi/2) = sin((k + 64) * pi/128).
-  DoubleDouble msin_k = SIN_K_PI_OVER_128[(k + 128) & 255];
+  DoubleDouble sin_k = SIN_K_PI_OVER_128[(k + 128) & 255];
```

Comment:
Can you change line 96 to `msin_k` instead?

Reply:
done